submissions/the-darkesttt/frogger-task/app.js

```diff
@@ -0,0 +1,323 @@
+const playerSkins = [
+    {
+        id: 'boy',
+        src: 'images/char-boy.png',
+    },
+    {
+        id: 'cat-girl',
+        src: 'images/char-cat-girl.png',
+    },
+    {
+        id: 'horn-girl',
+        src: 'images/char-horn-girl.png',
+    },
+    {
+        id: 'pink-girl',
+        src: 'images/char-pink-girl.png',
+    },
+    {
+        id: 'princess-girl',
+        src: 'images/char-princess-girl.png',
+    },
+];
+
+class Menu {
+    bg;
+    menu;
+    menuTitle;
+    menuDescriprion;
+
+    menuList;
+    menuItem;
+
+    menuScore;
+    menuClose;
+
+    constructor() {
+        this.bg = document.createElement('div');
+        this.bg.classList.add('bg');
+        this.menu = document.createElement('div');
+        this.menu.classList.add('menu');
+        this.menuTitle = document.createElement('h1');
+        this.menuTitle.innerText = 'Game Menu';
+        this.menuDescriprion = document.createElement('p');
+    }
+
+    start() {
+
+        this.menuDescriprion.innerText = 'Choose your character!';
+        this.menuList = document.createElement('ul');
+
+        playerSkins.forEach((item) => {
+            this.menuItem = document.createElement('li');
+    
+            const itemImg = document.createElement('img');
+            const imgSrc = item.src;
+            const imgId = item.id;
+            itemImg.setAttribute('src', imgSrc);
+            itemImg.setAttribute('id', imgId);
+            const imgDesc = document.createElement('p');
+            imgDesc.innerText = imgId;
+    
+            this.menuList.appendChild(this.menuItem);
+            this.menuItem.appendChild(itemImg);
+            this.menuItem.appendChild(imgDesc);
+            
+        });
+
+        document.body.appendChild(this.bg);
+        document.body.appendChild(this.menu);
+
+        this.menu.appendChild(this.menuTitle);
+        this.menu.appendChild(this.menuDescriprion);
+        this.menu.appendChild(this.menuList);
+
+        this.menuList.addEventListener('click', (event) => {
```

Comment:
Since you only use `target` property from the `event` class, you can get it by desctructuring here:
```js
this.menuList.addEventListener('click', ({ target }) => {
```

It works exactly like
```js
this.menuList.addEventListener('click', (event) => {
    const target = event.target
```